README.md

```diff
@@ -4,84 +4,89 @@ xhgui
 A graphical interface for XHProf data built on MongoDB.
 
 This tool requires that [XHProf](http://pecl.php.net/package/xhprof) or 
-its fork [Uprofiler](https://github.com/FriendsOfPHP/uprofiler) is installed, 
-which is a PHP Extension that records and provides profiling data.
+its fork [Uprofiler](https://github.com/FriendsOfPHP/uprofiler) is installed.
+XHProf is a PHP Extension that records and provides profiling data.
 XHGui (this tool) takes that information, saves it in MongoDB, and provides
-a convienent GUI for working with it.
-
+a convenient GUI for working with it.
 
 System Requirements
 ===================
 
+XHGui has the following requirements:
+
  * [XHProf](http://pecl.php.net/package/xhprof) or 
-   [Uprofiler](https://github.com/FriendsOfPHP/uprofiler) to actually profile the data
- * [MongoDB PHP](http://pecl.php.net/package/mongo) MongoDB PHP extension
- * [MongoDB](http://www.mongodb.org/) MongoDB Itself
- * [mcrypt] (http://php.net/manual/en/book.mcrypt.php) PHP must be configured
-   with mcrypt (which is a dependency of Slim)
- * [dom] (http://php.net/manual/en/book.dom.php) If you are running the tests
-   you'll need the DOM extension (which is a dependency of PHPUnit)
+   [Uprofiler](https://github.com/FriendsOfPHP/uprofiler) to actually profile the data.
+ * [MongoDB PHP](http://pecl.php.net/package/mongo) MongoDB PHP driver.
+   XHGui requires verison 1.3.0 or later.
```

Comment:
I suspect it's backward compatible, since the drivers team is pretty pro-that. BUT, worth double checking :)